src/gen_apple/ga_dev_gps.h

```diff
@@ -0,0 +1,22 @@
+#include <Arduino.h>
+#include <Adafruit_GPS.h>
+//#include <TinyGPS++.h>
+#include <SoftwareSerial.h>
+
+#define _PIN_GPS_RX 4
```

Comment:
I would change these macros to _GA_PIN so that it's more clear its for the apple generation.